ui/page/collection/index.js

```diff
@@ -5,7 +5,7 @@ import { makeSelectClaimForClaimId } from 'redux/selectors/claims';
 
```

Comment:
Not sure if legacy or not, but I tried to copy one of my public playlist and then tried to publish ... it says list is empty but the wallet shows items are correctly populated.

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/64950861/188501126-e40dac6a-3eca-4d90-9385-f1cc332d34e9.png">

Reply:
Not legacy -- changed to master and the publish form looks correct (allows me to publish)